src/sagemaker/session.py

```diff
@@ -195,17 +195,9 @@ def train(self, image, input_mode, input_config, role, job_name, output_config,
                     a directory in the Docker container.
                 * 'Pipe' - Amazon SageMaker streams data directly from S3 to the container via a Unix-named pipe.
 
-            input_config (str or dict or sagemaker.session.s3_input): Information about the training data.
-                This can be one of three types:
-
-                * (str) - the S3 location where training data is saved.
-                * (dict[str, str] or dict[str, sagemaker.session.s3_input]) - If using multiple channels for
-                    training data, you can specify a dict mapping channel names
-                    to strings or :func:`~sagemaker.session.s3_input` objects.
-                * (sagemaker.session.s3_input) - channel configuration for S3 data sources that can provide
-                    additional information about the training dataset. See :func:`sagemaker.session.s3_input`
-                    for full details.
-
+            input_config (list): An array of Channel objects. Each channel is a named input source. Please refer to
```

Comment:
Should be "A list of Channel objects". "Array" in python actually refers to a different type: https://docs.python.org/3/library/array.html

Reply:
@winstonaws Thanks. I fixed it in a7178c3.